src/Server/routes/SaveAllData.js

```diff
@@ -1,8 +1,11 @@
 const router = require('express').Router();
-
+const ModUSPlacesManager = require('../modules/USPlacesManager.js');
 router.post('/', (req, res) => {
 	console.log('Client session was closed');
 	// TO DO: Save all data in the respective csv files.
+	res.send({
+		success: ModUSPlacesManager.SaveRecords(),
```

Comment:
We don't need to send anything. I think we can just have SaveRecords() under the line console.log('Client session was closed'); :)